front-end/src/Home.jsx

```diff
@@ -9,13 +10,13 @@ function Home() {
     return (
         <div className='Home'>
             <header>
-                <Navbar username={username}/>
+                <Navbar username={username} />
             </header>
             <main>
                 <div className='beatTitle'>
-                    <h2>Your Beats</h2>
+                    <Heading as='h2' size='2xl'>Your Beats</Heading>
```

Comment:
Do you want to call each of these Beats or Stitches (from the app title)

Reply:
Ah yes, I will switch it! Love the idea thank you